pinot-plugins/pinot-file-system/pinot-adls/src/test/java/org/apache/pinot/plugin/filesystem/test/AzurePinotFSTest.java

```diff
@@ -74,8 +72,6 @@ public void testFS()
     Assert.assertTrue(azurePinotFS.exists(new URI(_adlLocation)));
 
     File file = new File(_adlLocation, "testfile2");
-    MockADLFileInputStream adlFileInputStream =
```

Comment:
Can you check the usage of `MockADLFileInputStream` ? If there's no usage, we can remove this class.

Reply:
True. I verified its not used and removed